src/wrappers/wrapper.js

```diff
@@ -120,6 +120,30 @@ export default class Wrapper implements BaseWrapper {
     return true
   }
 
+  /**
+   * Utility to check wrapper is visible. Returns false if a parent element has display: none or visibility: hidden style.
+   */
+  visible (): boolean {
+    if (!this.exists()) {
```

Comment:
You should check for an element here instead of exists—`if(!this.element)`. That's how we check for an element in other tests (I think this might solve the flow error)